__tests__/integration/atomic-swap.test.js

```diff
@@ -86,11 +88,13 @@ describe('send tx (HTR)', () => {
       .get('/wallet/atomic-swap/tx-proposal/get-locked-utxos')
       .set({ 'x-wallet-id': wallet1.walletId });
     loggers.test.insertLineToLog('atomic-swap[lock]: check with lock', { body: response.body });
+    // XXX: We have an extra utxo which should not be returned since its spent.
+    // This is not concerning but it means the selected utxos are not being correctly cleaned
```

Comment:
There is already a [PR](https://github.com/HathorNetwork/hathor-wallet-lib/pull/498) for this.
The reason its not concerning is that the utxos are not being selected, this is the list of utxos that cannot be selected.
The extra utxo here cannot be selected because it is spent, so it should not be here, but it will never be selected as an input either way.